src/dataframe/dataframe.jl

```diff
@@ -9,6 +9,7 @@ particularly a Vector, DataVector, or PooledDataVector.
 ```julia
 DataFrame(columns::Vector, names::Vector{Symbol})
 DataFrame(kwargs...)
+DataFrame(Pair{Symbol}...)
```

Comment:
Need `::` (and check that the result is OK locally by running `julia docs/make.jl`).